binding.js

```diff
@@ -20,9 +20,14 @@ var vipsHeaderPath = path.join(__dirname, 'include', 'vips', 'vips.h');
 var platform = process.env.npm_config_platform || process.platform;
 
 var arch = process.env.npm_config_arch || process.arch;
-if (arch === 'armhf') arch = 'arm';
-
-var arm_version = process.env.npm_config_armv || process.config.variables.arm_version;
+var arm_version = process.env.npm_config_armv || process.config.variables.arm_version || '7';
```

Comment:
The default value for `arm_version` is currently 6 - see https://github.com/lovell/sharp/blob/master/binding.js#L59